changelog/README.md

```diff
@@ -4,6 +4,13 @@
 
 * Added [Pagination](../guidelines/pagination.md) to [Get all resource access tokens](../operations/resourceaccesstokens.md#get-all-resource-access-tokens).
 
+## 5th October 2022
+
+* Added 'AddFeesToInvoices' to [Company Options](../operations/companies.md#company-options).
+* Added 'Department', 'DunsNumber' and 'CreditRating' field in [Company](../operations/companies.md#company).
+* Added [Credit Rating](../operations/company.md#credit-rating).
```

Comment:
Two broken links here (company.md should be companies.md) -

- [Credit Rating](../operations/company.md#credit-rating)
- [Credit Rating Basic](../operations/company.md#credit-rating-basic)

Reply:
@MikeAdamsMews Done